documentation/modules/retrieving-validation-service-json.adoc

```diff
@@ -19,6 +19,20 @@ You can create a validation rule based on any attribute in the `"input"` key, fo
 $ {oc} get route <inventory_service> -n {namespace}
 ----
 
+. Retrieve the access token:
++
+[source,terminal]
+----
+$ TOKEN = $(oc whoami -t)
+----
+
+. Trigger an HTTP GET request (for example, using Curl):
++
+[source,terminal]
+----
+$ curl -H "Authorization: Bearer $TOKEN" https://<inventory_service_route>/providers
```

Comment:
When I'm testing this command, I have to add -k, @ahadas, should it be added here?

Reply:
@qiyuann where do you add it?

Reply:
just after curl, curl -k -H...

Reply:
I don't remember having to use an insecure connection, but it could be that the reason for this was that I've triggered it from within the cluster, which is not a typical flow.. so that's ok to add the insecure flag